.github/workflows/SetUpAppForNuget.ps1

```diff
@@ -0,0 +1,70 @@
+[CmdletBinding()]
+param (
+    [switch]
+    $UseNuGet,
+    [Version]
+    $WinUIVersion = "2.6.0"
+)
+
+$AppName = Split-Path $PWD -Leaf
+Write-Host  App name = $AppName
+$ns = "http://schemas.microsoft.com/developer/msbuild/2003"
+
+[xml]$packagesConfig = Get-Content .\windows\$AppName\packages.config
+($packagesConfig.packages.package | Where-Object id -EQ 'Microsoft.UI.Xaml').version = $WinUIVersion.ToString()
+$packagesConfig.Save("$PWD\windows\$AppName\packages.config")
+
+[xml]$EF = Get-Content .\windows\ExperimentalFeatures.props
+$node = $EF.Project.PropertyGroup.WinUI2xVersion
+if ($null -eq $node) {
+  $node=$EF.CreateElement('WinUI2xVersion', $ns)
+  $v=$EF.CreateTextNode($WinUIVersion.ToString())
+  $node.AppendChild($v)
+  $EF.Project.PropertyGroup.AppendChild($node)
+} else {
+  $EF.Project.PropertyGroup.WinUI2xVersion = $WinUIVersion.ToString()
```

Comment:
Why do we need to override winui version?

Reply:
0.64 shipped with WinUI 2.3
react-native-xaml uses controls added in 2.6.